src/ontology/mondo-edit.obo

```diff
@@ -275865,11 +275862,9 @@ xref: UMLS:C1837091 {source="MEDGEN:373251", source="MONDO:equivalentTo", source
 is_a: MONDO:0018940 {source="DOID:0110679", source="MONDO:Redundant", source="OMIM:608931"} ! congenital myasthenic syndrome
 is_a: MONDO:0020344 {source="MONDO:Redundant", source="Orphanet:98913/btnt"} ! postsynaptic congenital myasthenic syndrome
 is_a: MONDO:1040021 {source="https://orcid.org/0000-0002-9698-992X"} ! congenital myasthenic syndrome 4
-intersection_of: MONDO:0018940 ! congenital myasthenic syndrome
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/1966 ! CHRNE
-intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4241 ! GFPT1
```

Comment:
Sabrina - removing the incorrect relationship `intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4241 ! GFPT1` then caused an [error with the qc check for omim subsumption](https://github.com/monarch-initiative/mondo/actions/runs/11716038830/job/32633442177#step:4:32). The reasoner was showing that the '4C' entry was a parent of '4A' and '4B', which did not look correct so that is partly why I removed the logical statement and moved the gene association relationship to the SubClassOf. There are some other ways this could be resolved, please let me know if this should be handled differently.